### ISSUE TITLE
Suggestion: Virtual Input name change

### ISSUE THREAD

**Issue by Dvogonen** [opentx/opentx#590]
The name virtual inputs is confusing. When it is shortened to only "input", like in the edit dialog it is very confusing. There are already a multitude of inputs in a radio:
- Movement of sticks and switches can be called inputs.
- The actual sticks and switches can be called inputs.
- Data coming in some kind of interface can be called input.
- The interface itself can be called an input.

When the user stumbles on yet another set pf inputs and the configuration of those, things will get complicated.

The most confusing aspect is that the virtual inputs are not really inputs at all.
They are instead inputs with some automatic calculations applied.
To me they are really filters.

In short I think that a better name for Virtual Inputs would simply be Filters.

The filter term captures what the actual functionality is.
It is unambiguous.
It is short.
It is translatable into every language I can think of (Input is terrible to translate into at least Swedish and German).
